core/state/database.go

```
@@ -706,6 +706,7 @@ func (tds *TrieDbState) updateTrieRoots(forward bool) ([]common.Hash, error) {
 				if len(v) > 0 {
 					//fmt.Printf("Update storage trie addrHash %x, keyHash %x: %x\n", addrHash, keyHash, v)
 					if forward {
+						_ = ClearTombstonesForNewStorage(tds.db, cKey)
```

Ideally this error should not be swallowed (can do in the next PR)